apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -133,6 +134,9 @@ public function normalize_theme_slug( $theme_slug ) {
 		if ( 'pub/' === substr( $theme_slug, 0, 4 ) ) {
 			$theme_slug = str_replace( 'pub/', '', $theme_slug );
 		}
+		if ( '-wpcom' === substr( $theme_slug, -6, 6 ) ) {
+			$theme_slug = preg_replace( '#-wpcom$#', '', $theme_slug );
```

Comment:
I wonder, from a purely performance standpoint, if it would be better to use a `substr( $theme_slug, 0, -6 )` instead of the regexp, considering we know both the string length and position.
(Same could be said for the previous `str_replace`, btw)